workflow/github.md

```diff
@@ -78,12 +78,12 @@ If we haven't written a styleguide for the language you're working in, please fe
 
 Here are the steps that need to be completed before merging:
 
+- If your proposal is still incomplete, you should at first create a [Draft Pull Request](https://github.blog/2019-02-14-introducing-draft-pull-requests/).
 - All [review labels](./labels.md#review) have to be at +1
 - Your pull request must be approved by at least one person
 - Any automated checks must have passed
-- The original author should merge a pull request
-  - This can be overruled when necessary, for example if all looks good and the changes are needed for further development and the author is AFK.
-  - Special care should be taken to ensure there have been no further changes requested
+- Special care should be taken to ensure there have been no further changes requested
+- The person who reviews your Pull Request will merge it once all checks and reviews have passed.
```

Comment:
This wording could be interpreted as forbidding the author from merging the PR themselves. I'm not sure that's quite right. It's also fine for you to merge it yourself. Often the final +1 might be a design +1 from a designer who isn't confident enough to click the "merge" button (although we should encourage it).

Could we say something like "Any team member may merge a pull request once it has met the above requirements."

Reply:
Or you could be a little more pushy with something like  "Any team member may merge a pull request once it has met the above requirements. This will usually be the person who added the final +1."

Reply:
Ok, updated to your latter suggestion.